ci/helpers.js

```diff
@@ -0,0 +1,88 @@
+const https = require('https')
+const { ECSClient, ListTasksCommand } = require('@aws-sdk/client-ecs')
+
+
+/**
+ * Returns list of running ECS Cloudwatch logs for running test tasks
+ * @param {Array<string>} taskArns 
+ * @returns {Array<string>}
+ */
+function generateDiscordCloudwatchLogUrls(taskArns) {
+  const arnRegex = /\w+$/
+
+  const LogUrls = taskArns.map((arn, index) => {
+    let LogUrlName 
+    const id = arn.match(arnRegex)
+    if (id) {
+      LogUrlName = `${process.env.CLOUDWATCH_LOG_BASE_URL}${id[0]}`
+    }
+
+    return `${LogUrlName}\n`
+  })
+
+  return LogUrls 
+}
+
+
+/**
+ * Returns list of running ECS anchor tasks
```

Comment:
Update this comment because they are not anchor tasks
```suggestion
 * Returns list of running ECS tasks
```

Reply:
Updated